src/store.js

```diff
@@ -18,6 +18,7 @@ export const initialState = {
     currentPreviewResource: undefined,
     currentComponent: {},
     currentModal: undefined,
+    currentLangModalValue: undefined, // the value key of the value to be displayed in the InputLang modal.
```

Comment:
"value key"?   vs "key"?

Reply:
Yeah, I thought about this, but dropping the "key" is consistent with the naming conventions in the editor section of the store.